src/mumble/Settings.h

```diff
@@ -369,6 +369,7 @@ struct Settings {
 
 	QPoint qpTalkingUI_Position              = UNSPECIFIED_POSITION;
 	bool bShowTalkingUI                      = false;
+	bool bTalkingUI_UsersAlwaysVisible       = false;
```

Comment:
Ok, one last thing (sorry). Could you please remove the type prefix "b" here? This is one of the new Mumble style requirements. Should have noticed earlier, but I am usually of the other side of code reviews ;)

Reply:
No problem I fixed it now ;)